SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialGameInstance.cpp

```diff
@@ -70,7 +70,7 @@ void USpatialGameInstance::CreateNewSpatialWorkerConnection()
 	SpatialConnection = NewObject<USpatialWorkerConnection>(this);
 }
 
-FORCEINLINE USpatialWorkerConnection* USpatialGameInstance::GetSpatialWorkerConnection()
+USpatialWorkerConnection* USpatialGameInstance::GetSpatialWorkerConnection()
```

Comment:
Is this intended?

Reply:
This will go away after I rebase the PR on the hydra master. Currently validating the rebase.